src/graph/Graph.js

```diff
@@ -694,6 +694,9 @@
                     }
                 }
             })
+            .each(function(d){
+                d3.select(this).classed("graphVertex-highlighted",!vertexMap || vertexMap[d.id()]);
+            })
             .style("opacity", function (d) {
```

Comment:
Your combination of "each" and "d3.select(this)" can be avoided as its the same as:
```
    .classed("graphVertex-highlighted", function (d) { return !vertexMap || vertexMap[d.id()]; })
```
I would suggest moving it above the transition:
```
        vertexElements
            .classed("graphVertex-highlighted", function (d) { return !vertexMap || vertexMap[d.id()]; })
            .transition().duration(this.transitionDuration())
            .each("end", function (d) {
                if (vertexMap && vertexMap[d.id()]) {
                    if (d._parentElement.node() && d._parentElement.node().parentNode) {
```